index.html

```diff
@@ -5650,6 +5642,38 @@ <h2 id="defining-a-valid-audioworkletprocessor">
               This in turn can affects whether subsequent invocations of
               <code>process()</code> occur, via the condition cited above.
```

Comment:
This wasn't part of this change, but maybe we should have a link for "condition cited above".  There are many conditions  above, so let's be specific.